code/__HELPERS/text.dm

```diff
@@ -1188,3 +1188,13 @@ GLOBAL_LIST_INIT(binary, list("0","1"))
 /proc/sanitize_css_class_name(name)
 	var/static/regex/regex = new(@"[^a-zA-Z0-9]","g")
 	return replacetext(name, regex, "")
+
+/// Returns a or an depending on what the text starts with.
+/proc/prefix_a_or_an(text)
```

Comment:
I feel like this is better described as a vowel check, so someone doesn't write one again. 

/proc/piglatin_word(word) - Exists already and basically does this, for example, so yeah, just thinking of duplicate code in the future showing up when it can be kinda unified for future use purposes.

Reply:
`piglatin_word` does not do this at all, that is for applying a sol accent to a sentence spoken ICly. Not that it's actually ever used AFAIK.

`a_or_an` makes the most sense, cause it's in plain english, and is pretty to the point.